### PR TITLE
Update vcpkg to current version

### PR DESCRIPTION
**Pull request by janisozaur**
Updates:
- benchmark 1.6.1 -> 1.7.1
- breakpad 2020-09-14 -> 2022-07-12
- discord-rpc 3.4.0 (no change)
- freetype 2.11.1 -> 2.12.1
- fribidi 1.0.12 (no change)
- gtest 1.11.0 -> 1.13.0
- libflac 1.3.4 -> 1.4.2
- libogg 1.3.5 (no change)
- libvorbis 1.3.7 (no change)
- libpng 1.6.37 -> 1.6.39
- libzip 1.8.0 -> 1.9.2
- nlohmann-json 3.10.5 -> 3.11.2
- openal-soft 1.21.1 -> 1.23.0
- SDL 2.0.22 -> 2.26.4
- speexdsp 1.2.0 -> 1.2.1
- zlib 1.2.12 -> 1.2.13

### CODE REVIEW

**Comment by janisozaur** [comment]
Filed https://github.com/microsoft/vcpkg/issues/30532 for macOS failure

**Review by tupaschoal — COMMENTED** [review]
SDL is quite a jump, I wonder whether it will make any difference

**Comment by janisozaur** [comment]
They changed their versioning scheme

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: CI
 on: [push, pull_request]
 env:
-  VCPKG_COMMIT_HASH: 5e7cbdceacd2b3cb85e057963fdb605136805bd3
+  VCPKG_COMMIT_HASH: 23f0707b1a46bbf7fff9fb95cde2aa0c7213c31d
```
Comment (janisozaur) [inline .github/workflows/ci.yml:4]:
```suggestion
  VCPKG_COMMIT_HASH: c9f906558f9bb12ee9811d6edc98ec9255c6cda5
```

**Pull request merged**